index.js

```diff
@@ -3,14 +3,13 @@
 const { Unauthorized, InternalServerError } = require('http-errors')
 const fastifyPlugin = require('fastify-plugin')
 const fastifyJwt = require('fastify-jwt')
-const jwt = require('jsonwebtoken')
 const fetch = require('node-fetch')
 const NodeCache = require('node-cache')
 
 const forbiddenOptions = ['algorithms']
 
 const errorMessages = {
-  badHeaderFormat: 'Authorization header should be in format: Bearer [token].',
+  badHeaderFormat: 'Authorization header should be in format: Bearer [token].', // TODO: to remove
```

Comment:
Is this message still used?

Reply:
yep, this error message are used, going to remove wrong todos